app/assets/javascripts/form_exit.js.erb

```diff
@@ -5,10 +5,20 @@
     if(outsideOfSteps) return;
 
     for(var i=0; i<links.length; i++){
-      var stepsLink = links[i].href.indexOf('steps') >= 0
-      var skipLink = links[i].href.indexOf('skip_') >= 0
+      var isAnExternalLink = true
+      var whiteList = [
+        '/privacy',
+        '/steps',
+        '/terms',
+        'skip_'
+      ]
+      for(var j=0; j<whiteList.length; j++){
```

Comment:
style question: in JS do you like to keep no spaces or would it be better to have `j = 0; j < whiteList.length`? Can't tell if ruby style is infecting my brain here

Reply:
It would be nice if we had a rubocop for js set up to make these decisions for us.